suite/tests/CMakeLists.txt

```diff
@@ -1946,6 +1946,12 @@ if (CLIENT_INTERFACE)
         "" "${trace_arg}" "${events_appdll_path}")
       tobuild_ci(client.nudge_test client-interface/nudge_test.runall "" "" "")
       tobuild_ci(client.timer client-interface/timer.c "" "" "")
+      tobuild_ci(client.mangle_suspend client-interface/mangle_suspend.c ""
```

Comment:
Seems like this should be 64-bit-only for x86, right?  It's a nop in 32-bit x86 and just wastes Travis resources?

Reply:
Added if (X64). Thanks.